services/worker/src/worker/job_runners/split/opt_in_out_urls_scan_from_streaming.py

```diff
@@ -125,26 +128,25 @@ async def check_spawning(
 
 async def opt_in_out_task(
```

Comment:
maybe this function is no more useful, and the try/except could go to the check_spawning function?

Reply:
Or move the checks below, to the loop where you use the contents of spawning_response